scripts/recipes/bread_blocks.zs

```diff
@@ -0,0 +1,13 @@
+print ("Patching bread block recipes");

```

Comment:
probably don't need these prints tbh

Reply:
That's a convention from my own scripting, so I know where each script begins and ends in the log. I'll remove them if you want though.

Reply:
I don't really have a strong opinion, feel free to leave them in if you want. We just don't currently use them anywhere else.